app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -3,28 +3,45 @@
 @import '../../../sizes'
 
 table
+  display: grid
+  grid-template-columns: 1fr min-content min-content min-content min-content
   border: solid 1px $maincolor
   border-collapse: collapse
+
+
+
   thead
+    display: contents
     border: solid 1px $maincolor
-  td, th
+  tbody,tr
+    display: contents
+  tr:hover td
+    background-color: $softbgcolor
```

Comment:
Please move these two lines close to the part about td. You could even leave that rule where it was, just replacing `&:hover` with `&:hover td`.